apps/wpcom-block-editor/src/wpcom/features/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Determines the type of the block editor.
+ *
+ * @returns {'post'|'site'|undefined} editor's type
```

Comment:
I think it's not a valid JSDoc format. It would probably require i.e. an EditorType `@typedef`. Otherwise I think it should be:
```suggestion
 * @returns {string|undefined} The editor's type
```